src/prune_builds

```diff
@@ -15,6 +15,9 @@ import subprocess
 
 import dateutil.parser
 
+sys.path.insert(0, '/usr/lib/coreos-assembler')
```

Comment:
Oh weird, I thought it would already know to check the same dir as the script.

Reply:
actually yeah.. at times I've copied the contents of `/usr/lib/coreos-assembler` (read-only in some setups I've been using) to another directory and modified things slightly and then re-ran. could we just insert the path to wherever this file is?

Reply:
IMO Dusty this is one of those things where submitting a PR to do that would have been less typing :smile: